tests/test_views.py

```diff
@@ -4359,5 +4363,92 @@ def test_confirm_user_signal_called_when_user_confirms_email(self):
 
         assert_equal(mock_signals.signals_sent(), set([auth.signals.user_confirmed]))
 
+
+# copied from tests/test_comments.py
+class TestCommentViews(OsfTestCase):
+
+    def setUp(self):
+        super(TestCommentViews, self).setUp()
+        self.project = ProjectFactory(is_public=True)
+        self.user = AuthUserFactory()
+        self.project.add_contributor(self.user)
+        self.project.save()
+        self.user.save()
+
+    def test_view_project_comments_updates_user_comments_view_timestamp(self):
+        url = self.project.api_url_for('update_comments_timestamp')
+        res = self.app.put_json(url, {
+            'page': 'node',
+            'rootId': self.project._id
+        }, auth=self.user.auth)
+        self.user.reload()
+
+        user_timestamp = self.user.comments_viewed_timestamp[self.project._id]
+        view_timestamp = dt.datetime.utcnow()
```

Comment:
Minor: we should use django.utils.timezone.now instead of utcnow to mimic how datetimes are stored in app code. Not a big deal here though.